### DISCUSSION TITLE
🏆 Open Source Heroes

### DISCUSSION THREAD

**Discussion by markets** [therubyradar/submissions#184] [Submissions]
Hello 👋🏼 

Open Source Heroes is a platform to explore the open source ecosystem data and trends: [https://opensource-heroes.com](https://opensource-heroes.com/). This is not only for Ruby projects, but the system is built with: Rails, PostgreSQL, Redis, Meilisearch, Tailwind and Turbo/Hotwire.

You can find different kind of lists, rankings and collections. Some examples:

* Top Ruby [Contributors](https://opensource-heroes.com/contributors?language=ruby): with filters by Language and/or Country
* [Trending](https://opensource-heroes.com/discover/ruby) Ruby: most starred repos with different date ranges (30 days, 90 days, All time)
* [Awesome](https://opensource-heroes.com/awesome/ruby) Ruby: with extra info like stars, last update and license
* Embeddable (for your README or your site) stars history for each repo
* [Country/city stats](https://opensource-heroes.com/countries): top contributors, cities with more open source contributors, top rank for each language, compare with other countries
* "Made in": a page for each country, with Top 100 repos (also filterable by lang, so you can filter out things like: [Top Ruby projects from Spain](https://opensource-heroes.com/made-in/spain?language=Ruby))

**Comment by cjilbert504** [comment]
@markets - Thanks!
